dev/com.ibm.ws.config/resources/com/ibm/ws/config/internal/resources/ConfigMessages.nlsprops

```diff
@@ -460,6 +460,10 @@ info.config.conflict=CWWKG0102I: {0}
 info.config.conflict.explanation=Multiple values are specified for the same configuration property. 
 info.config.conflict.useraction=No action is required. The informational message can be avoided by editing the server configuration to remove the conflict.
  
+warning.supplied.config.not.valid=CWWKG0103W: The attribute name {0} with value {1} will be ignored because it is defined as ibm:final. The system will use the default value of the attribute.
```

Comment:
We try to stick with present tense if at all possible. So, for example, we'd say, "...value is ignored..." and "The system uses the default..."

However, I'm not seeing the message in context, so if it present tense is weird here, we can leave the future tense as-is. :)